### DISCUSSION TITLE
Unable to convert PDF to Image on Azure Web App.

### DISCUSSION THREAD

**Discussion by ghost** [dlemstra/Magick.NET#715] [Help]
Hi,

So I have installed the "Ghostscript" on my local system and also using the "Magick.NET-Q16-HDRI-AnyCPU" library. I am able to convert a PDF page to image and then save it on filesystem without any problems. 

Now, I have a trial MS Azure web app. I deployed app. to it, copied the "gsdll64.dll" and "gswin64c.exe" files to "bin" directory of server manually, but still I'm unable to use the feature. 

I get the following error back: 

`FailedToExecuteCommand `\"gswin64c.exe\" -q -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT -dMaxBitmap=500000000 -dAlignToPixels=0 -dGridFitTT=2 \"-sDEVICE=pngalpha\" -dTextAlphaBits=4 -dGraphicsAlphaBits=4 \"-r300x300\"  \"-sOutputFile=D:/local/Temp/magick-mM-HSlTmDz7_Z8rVKhBWUeYzBZtp7PII%d\" \"-fD:/local/Temp/magick-r2_fg4WniyceIms16PRfXLaOcM9B0Hx_\" \"-fD:/local/Temp/magick-mdVxOPhKsVAF67FlIgTGDYVgAifJLspA\"' (The system cannot find the file specified.\r\n) @ error/delegate.c/ExternalDelegateCommand/475"
`

There is a mention of "TEMP" directory here, not sure about that? **Please someone help me here. I'm on a strict deadline and have to get this done quick! Any thoughts would be appreciated, thank you!!!**

**Comment by dlemstra** [comment]
The message "The system cannot find the file specified." could also indicate that the system is unable to find the `gswin64c.exe` file. My advise would be to copy the binaries to an extra folder in the project and use that folder as an argument for the `MagickNET.SetGhostscriptDirectory()` method.
